go-controller/pkg/ovn/gress_policy.go

```diff
@@ -382,17 +384,45 @@ func (gp *gressPolicy) modifyACLAllow(oldMatch, newMatch string) error {
 	return nil
 }
 
+func constructAclIPString(direction string, ipCIDRs []string) string {
```

Comment:
"ACL" not "Acl". Although having two acronyms back-to-back ("ACLIP") is awkward so maybe try to rename

Reply:
Done